packages/components/psammead-story-promo-list/src/index.jsx

```diff
@@ -36,8 +37,12 @@ const StyledListItem = styled.li`
 
 const StyledUnorderedList = styled.ul`
   list-style-type: none;
-  margin: 0;
+  margin: ${GEL_SPACING} 0 0 0;
```

Comment:
Is UX happy with this separation if there is no `SectionLabel`. 
At the moment `SectionLabel` uses a margin-top of `1.5rem`, but you won't be able to overlap with that value.

Reply:
Yep, 16px and 8px to match the side padding